docs/trulens_eval/feedback_functions_existing_data.md

```diff
@@ -0,0 +1,177 @@
+In many cases, developers have already logged runs of an LLM app they wish to evaluate or wish to log their app using another system. Feedback functions can also be run on existing data, independent of the `recorder`.
+
+At the most basic level, feedback implementations are simple callables that can be run on any arguments
+matching their signatures like so:
+
+```python
+feedback_result = provider.relevance("<some prompt>", "<some response>")
+```
+
+!!! note
+
+    Running the feedback implementation in isolation will not log the evaluation results in TruLens.
+
+In the case that you have already logged a run of your application with TruLens and have the record available, the process for running an (additional) evaluation on that record is by using `tru.run_feedback_functions`:
+
+```python
+tru_recorder = TruChain(
+    chain,
+    app_id='Chain1_ChatApplication'
+)
+
+with tru_recorder as recording:
+    record = chain(""What is langchain?")
+
+tru.run_feedbacks(record, feedbacks=[f_lang_match, f_qa_relevance, f_context_relevance])
+```
+
+### TruVirtual
+
+If your application was run (and logged) outside of TruLens, `TruVirtual` can be used to ingest and evluate the logs.
+
+To incorporate your data into TruLens, you have two options. You can either create a `Record` directly, or you can use the `VirtualRecord` class, which is designed to help you build records for virtual models.
```

Comment:
I don't know if this will confuse users, use something like, "designed help you build records without actually running or calling the generative model API"